test/index.test.js

```diff
@@ -980,4 +980,36 @@ describe('index test', () => {
                 });
         });
     });
+
+    describe('_openPr', () => {
```

Comment:
test is failing because you need to mock this function

Reply:
thanks I fixed it